<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1508</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qwirkel</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1491</width>
     <height>621</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1491</width>
     <height>621</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1469</width>
      <height>599</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1471</width>
       <height>601</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="lfeld">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="unten_links">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>641</width>
     <height>161</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>641</width>
     <height>161</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="hand" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="lhand">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Benutzerhand</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Punktestand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Regeln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="einstellungen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="unten_mitte">
   <property name="geometry">
    <rect>
     <x>649</x>
     <y>639</y>
     <width>521</width>
     <height>161</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>521</width>
     <height>161</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QListView" name="chatView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>521</width>
      <height>131</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>521</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>521</width>
      <height>27</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="unten_rechts">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>640</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>321</width>
     <height>161</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Tauschen" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 158, 158);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>tauschen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Beenden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Farbe?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tauschen</class>
   <extends>QFrame</extends>
   <header location="global">tauschen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
